dashboard/src/components/QuickNav.js

```diff
@@ -8,9 +8,9 @@ import DashboardIcon from "material-ui-icons/Dashboard";
 import EventIcon from "material-ui-icons/Notifications";
 import EntityIcon from "material-ui-icons/DesktopMac";
 import CheckIcon from "material-ui-icons/AssignmentTurnedIn";
-import SilenceIcon from "material-ui-icons/VolumeOff";
-import HookIcon from "material-ui-icons/Link";
-import HandlerIcon from "material-ui-icons/CallSplit";
+// import SilenceIcon from "material-ui-icons/VolumeOff";
+// import HookIcon from "material-ui-icons/Link";
+// import HandlerIcon from "material-ui-icons/CallSplit";
```

Comment:
Did you want to leave these commented out? These links still currently exist in the drawer.

Reply:
removed